javascript/app-framework/src/components/tables/ImportTable/ImportTable.tsx

```diff
@@ -45,6 +53,14 @@ export const ImportTable: React.FC<ImportTableProps> = memo(
 
 		const [name, setName] = useState<string>(removeExtension(file.name ?? ''))
 
+		const [rawContent, setRawContent] = useState<string | undefined>()
+		useEffect(() => {
+			const f = async () => {
```

Comment:
This short callback may be clearer as a promise chain

`file.toText().then(setRawContent)`